etna/models/prophet.py

```diff
@@ -135,10 +134,7 @@ def predict(self, df: pd.DataFrame, prediction_interval: bool, quantiles: Sequen
             DataFrame with predictions
         """
         df = df.reset_index()
```

Comment:
Why didn't you put this line inside the `_prepare_prophet_df `? You can add this line in fit too